.github/workflows/tests.yml

```
@@ -0,0 +1,69 @@
+name: Tests
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    paths:
+      - setup.py
+      - pyproject.toml
+      - MANIFEST.in
+      - tests/**
+      - safety_gymnasium/**
+      - .github/workflows/tests.yml
+  # Allow to trigger the workflow manually
+  workflow_dispatch:
+
+permissions:
+  contents: read
+
+concurrency:
+  group: "${{ github.workflow }}-${{ github.ref }}"
+  cancel-in-progress: ${{ github.event_name == 'pull_request' }}
+
+jobs:
+  test-ubuntu:
+    name: Test for Python ${{ matrix.python-version }} on ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest]
+        python-version: ["3.8", "3.9", "3.10"]
```

Don't we support Python 3.11?

```suggestion
        python-version: ["3.8", "3.9", "3.10". "3.11"]
```

---

In previous tests, Python 3.11 also had some errors.